### DISCUSSION TITLE
Set Formatting to Selection Similar to VScode

### DISCUSSION THREAD

**Discussion by jonmahoney15** [LazyVim/LazyVim#4649] [Q&A] [answered]
Instead of formatting on save, I would like to only format in visual mode what has been highlighted when I execute a given keymap.

I am pretty new to nvim and was hoping someone could help me figure out this keymapping?

**Comment by jonmahoney15** [comment]
I am not sure conform supports anything other than formatting an entire file.

**Reply by rogeriofrsouza** [reply, marked as answer]
Yes, it's supported. You can format in Visual Mode with the same keymap `<leader>cf`

**Reply by jonmahoney15** [reply]
wow this was a dumb question, thank you.
